api/src/utils/media/csv/validation/csv-row-validator.ts

```diff
@@ -223,6 +223,62 @@ export const getValidRangeFieldsValidator = (config?: ColumnRangeValidatorConfig
   };
 };
 
+export type ColumnNumericValidatorConfig = {
+  columnName: string;
+  column_numeric_validator: {
```

Comment:
some of these is snake_case and some are camelCase... can we standardize?

Reply:
We can, but lets do that in a separate PR.  Technically that is my doing from when I originally set it all up.